README.md

```diff
@@ -1,2 +1,28 @@
 # LinQuadOptInterface.jl
-Intermediate wrapper for MOI of GLPK, CPLEX, Gurobi and Xpress
+
+LinQuadOptInterface (LQOI) is designed to make it easier for low-level wrapper designed to bridge low-level Integer Linear and Quadratic solvers to implement the MOI interface. It provides access to many MathOptInterface functionalities mainly related to problem modifications:
+
+1. add constraints/variables 1-by-1 and in batches
+2. remove constraints/variables
+3. change constraint coefficients
+4. change constraint type
+5. change constraint rhs
+6. change variable bounds and type
+
+## LinQuadOptInterface Instance
+
+In LinQuadOptInterface.jl the MOI `AbstractOptimizer` is specified to `LinQuadOptimizer`. In this implementation all the above metioned modifications are caried out by the low-level solver own functionalities and hence it differs from MathOptInterface.Utilities' `AbstractModel`'s generated by the `@model` macro. The latter will keep all problem data in the julia level and typically push to a low-level solver the complete problem at once.
```

Comment:
metioned -> mentioned

Reply:
You mention @model but the discussion is closen ti CachingOptimizer

Reply:
Do you have any suggestion about what to say there or should I just remove this part?

Reply:
You added it when SDOI was keeping a copy but now no optimizer keeps a copy, `CachingOptimizer` should be used for that. You can say "In this implementation all the above mentioned modifications are carried out by the low-level solver own functionalities and hence the optimizer can be used without a `CachingOptimizer`"